src/components/ColorPalette/constants/color-api.js

```diff
@@ -0,0 +1,107 @@
+export const PALETTE = [
```

Comment:
```suggestion
export const PALETTES = [
```
gua gatau versi jamaknya apa bener ga tambah 's'? wkwk

Reply:
iya ko harusnya pake s ko hehe